approval_polls/models.py

```diff
@@ -1,52 +1,103 @@
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.contrib.auth.models import User
 
+from django.utils import timezone
+
+
 class Poll(models.Model):
-  question = models.CharField(max_length=200)
-  pub_date = models.DateTimeField('date published')
-  user = models.ForeignKey(User)
+    '''A Poll class. This is the top level container for a Poll.
+
+    referenced by Choices model and the Ballot model
+    '''
+
+    question = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+    user = models.ForeignKey(User)
+    open_date = models.DateField()
+    close_date = models.DateField()
+    suspended = models.BooleanField(default=False)
+
+    def is_closed(self):
+        '''Determine if the poll is open or closed
+
+        Arguments:
+            None
+
+        Returns:
+            boolean: True if the close_date < datetime.datetime.now()
```

Comment:
I disagree. Here is why. 

At some point you are going to want to have auto generated documentation probably using Sphinx. It is these doc strings which will at the very least make the core of the documentation which can be expanded if needed. 

Also, in the python command line shell and in the tool ipython there is the help() function. 

The help function returns the docstring defined for the object. Be it a class, function or method. This is very handy for someone working on or extending the code. 

While the method name is_closed seems explicit the method name alone does not make clear the criteria for the value returned. In other words we know that it is closed but we don't know why!

The documentation standard for python is PEP0257 https://www.python.org/dev/peps/pep-0257/ which is the current cannon.

Be that as it may I am actually using a variant of that developed by Google, which can be parsed out of the code by using the napoleon extension for Sphinx. 

http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html